webapp/apps/btax/compute.py

```diff
@@ -0,0 +1,57 @@
+import dropq
+import os
+from ..taxbrain.models import WorkerNodesCounter
+import json
+import requests
+from requests.exceptions import Timeout, RequestException
+from .helpers import arrange_totals_by_row
+from ..taxbrain.compute import (DropqCompute,
+                                MockCompute,
+                                MockFailedCompute,
+                                NodeDownCompute,
+                                JobFailError,
+                                ENFORCE_REMOTE_VERSION_CHECK,
+                                TIMEOUT_IN_SECONDS,
+                                dropq_version)
+import requests_mock
+requests_mock.Mocker.TEST_PREFIX = 'dropq'
+
+
+
+def package_up_vars(self, user_mods, first_budget_year):
```

Comment:
not sure what's happening here. free-standing function with the first argument `self` and `self` is not used in the function?